plugins/cue_points/base/lib/kCuePointManager.php

```diff
@@ -219,7 +219,11 @@ protected function deleteCuePoints(Criteria $c)
 		CuePointPeer::setUseCriteriaFilter(true);
 
 		foreach($cuePoints as $cuePoint)
+		{
+			$cuePoint->reload();
```

Comment:
What for? it's already deleted in the doUpdate.
However, if you want to reload them all, do it using retrieveByPKs in order to do it in one command in the DB.